cEP-0010.md

```diff
@@ -0,0 +1,150 @@
+# Convert bears to use aspects
+
+| Metadata     |                                                |
+| ------------ |------------------------------------------------|
+| cEP          | 10                                             |
+| Version      | 1.0                                            |
+| Title        | Convert bears to use aspects                   |
+| Authors      | Asnel Christian Ngoulla <ngoullasob@gmail.com> |
+| Status       | Implementation Due                             |
+| Type         | Feature                                        |
+
+## Abstract
+
+This document describes how `aspects` will be defined for and connected with
+bears and how their results will be annotated with them.
+
+This cEP also provides a list of the `aspects` to be implemented with their
+correspponding `tastes` or `settings`.
+
+## Introduction
+
+An `aspect` is an analysis we want to run over a piece of code. As stated in
+[cEP-0005](...), one of the main problems coala faces is its configuration due
+to the huge number of bears it offers, which are capable of analyzing several
+aspects of code and are sometimes highly configurable with numerous settings;
+whereby the user has to be aware of which bear does what and which setting has
+be to set to anylize the aspect he wants analyzed; resulting in a coala
+difficult to learn and use.
+
+To make things easier for user the concept of `aspects` was created as proposed
+in [[cEP-0005](...). Our goal here is to list all the aspects for covering all
+the code analysis features the coala bears are capable of, annotating bears'
+results with them.
+
+## The aspests
+
+The bears offered by coala are usually capable of handling several aspects.
+`aspects` will be implemented in a tree-like form starting with the
+[Root aspect](https://github.com/coala/coala/blob/master/coalib/bearlib/aspects/root.py).
+Here is the list of aspects to be implemented along with their corresponding
+`subaspects` and tastes or settings(which are the leaves of the tree written
+in snake_case).
+
+
+```
+  Root
```

Comment:
I would want a description of the newly defined aspects in this cEP. What kind of changes are the bears under them expected to offer.